### DISCUSSION TITLE
formily-editor演示长什么样的很好奇 在线访问js报错了

### DISCUSSION THREAD

**Discussion by tender001** [alibaba/formily#1283] [Ideas]
![image](https://user-images.githubusercontent.com/12458462/115822524-44d68c80-a437-11eb-99bc-ba800617f5dc.png)

**Comment by aspirantzhang** [comment]
把依赖中的antd改为v1版本就可以访问了。
![QQ截图20210423143313](https://user-images.githubusercontent.com/24559988/115828934-e9110100-a440-11eb-9d70-6bd55e9c2bac.jpg)
![QQ截图20210423143321](https://user-images.githubusercontent.com/24559988/115828946-ec0bf180-a440-11eb-96f1-3a4a5425bfc3.jpg)

**Reply by Z-Champion** [reply]
Thanks
